config/schedule.yml

```diff
@@ -9,3 +9,19 @@ weekly_metrics:
 merge_time_generator:
   cron: '00 23 * * *'
   class: 'ProcessMergeTimeMetricsJob'
+
+blog_posts_partial_update:
+  cron: '00 4 * * 3'
+  class: 'BlogPostsPartialUpdaterJob'
+
+blog_posts_full_update:
+  cron: '00 2 1 * *'
+  class: 'BlogPostsFullUpdaterJob'
+
+blog_metrics_partial_update:
+  cron: '30 4 * * *'
+  class: 'BlogMetricsPartialUpdaterJob'
+
+blog_metrics_full_update:
+  cron: '00 3 1 * *'
+  class: 'BlogMetricsFullUpdaterJob'
```

Comment:
Add active_job: true at the end of every cron job, eg: 
```
blog_posts_partial_update:
  cron: '00 4 * * 3'
  class: 'BlogPostsPartialUpdaterJob'
  active_job: true
```
And as a boy scout add that to the other ones :)

Reply:
Sure, no problem!
Why is that for?

Reply:
Sidekiq-cron gem documentation

![image](https://user-images.githubusercontent.com/37642427/83658765-0757df00-a599-11ea-8472-f5935b7b7b40.png)

We are using active job interface in our proyect.